iconrpcserver/server/rest_server.py

```diff
@@ -18,22 +18,21 @@
 from http import HTTPStatus
 from urllib.parse import urlparse
 
+from iconcommons.icon_config import IconConfig
+from iconcommons.logger import Logger
 from sanic import Sanic, response
-from sanic.views import HTTPMethodView
 from sanic.log import LOGGING_CONFIG_DEFAULTS
+from sanic.views import HTTPMethodView
+from sanic_cors import CORS
 
-from iconcommons.icon_config import IconConfig
-from ..default_conf.icon_rpcserver_constant import ConfigKey, NodeType, SSLAuthType
-from ..components import SingletonMetaClass
 from .peer_service_stub import PeerServiceStub
 from .rest_property import RestProperty
-from iconrpcserver.dispatcher.default import NodeDispatcher, WSDispatcher
-from iconrpcserver.dispatcher.v2 import Version2Dispatcher
-from iconrpcserver.dispatcher.v3 import Version3Dispatcher
-from iconrpcserver.dispatcher.v3d import Version3DebugDispatcher
+from ..components import SingletonMetaClass
+from ..default_conf.icon_rpcserver_constant import ConfigKey, SSLAuthType
+from ..dispatcher.default import NodeDispatcher, WSDispatcher
+from ..dispatcher.v3 import Version3Dispatcher
+from ..dispatcher.v3d import Version3DebugDispatcher
```

Comment:
Can you explain why do you change import statements from absolute to relative?

Reply:
Both explicit relative and absolute imports are allowed in PEP8, but I've changed them for consistency because I thought it's weird to mix two kinds of imports in one file.